### PR TITLE
CustomCard component

### PR DESCRIPTION
**Pull request by MartinCaparros**
Card created, its a component that recieves 3 props (title,img,description), in case that img its null it renders a placeholder.

User story: https://alkemy-labs.atlassian.net/browse/OT91-60

Evidence:

Captura de pantalla 2021-11-03 113754

### CODE REVIEW

File: src/Components/Card/CustomCard.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import {maxWidth,display} from '@material-ui/core'
+
+const CustomCard = ({title,img,description}) => {
+
+    return (
+        <Container sx={{ display: 'inline-flex' }}>
+            <Card sx={{ maxWidth: 345 }}>
```
Comment (saralavanchy) [inline src/Components/Card/CustomCard.js:15]: the styles should come from Styles folder. Its not a blocking but is a note for next PR.

File: src/Components/Card/CustomCard.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                <CardMedia
+                component="img"
+                height="140"
+                image={img? img.image : "https://via.placeholder.com/150"}
+                alt="Organization image"
+                />
+                <CardContent>
+                    <Typography gutterBottom variant="h5" component="div">
+                        {title? title : ""}
```
Comment (saralavanchy) [inline src/Components/Card/CustomCard.js]: you can use title || "" instead of title ? title : ""

File: src/Components/Card/CustomCard.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                    </Typography>
+                    <Typography variant="body2" color="text.secondary">
+                        {description? description : ""}
```
Comment (saralavanchy) [inline src/Components/Card/CustomCard.js]: same of title

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Components/Card/CustomCard.js (hunk 1 above)
Comment (MartinCaparros) [reply src/Components/Card/CustomCard.js:15]: Perfect, ill be aware of this on future PR.

File: src/Components/Card/CustomCard.js (hunk 2 above)
Comment (MartinCaparros) [reply src/Components/Card/CustomCard.js]: Already change it

File: src/Components/Card/CustomCard.js (hunk 3 above)
Comment (MartinCaparros) [reply src/Components/Card/CustomCard.js]: Already change it.

**Review by saralavanchy — APPROVED** [review]

**Review by saralavanchy — APPROVED** [review]

**Pull request merged**